extras/ffmpeg-decoder/src/ffmpeg/avcodec.rs

```diff
@@ -63,6 +63,7 @@ pub struct AvCodec(&'static ffi::AVCodec);
 
 /// SAFETY: `AVCodec` is static and thus safe to share.
 unsafe impl Send for AvCodec {}
+unsafe impl Sync for AvCodec {}
```

Comment:
IIUC implementing `Sync` is safe because none of the methods of `AvCodec` can ever change its state. Let's add that as a safety comment.